pan-Draft revision

nicola-debernardini opened the pull request:
Dear Johannes,

I have made some corrections regarding the following points:

1. Revised the 'gapseq -v' option recently added to print the md5 hash of the sequence database. I noticed that if you have never run the './src/update_sequences.sh' script it lacks the ./dat/seq/rev folder so it won't print the date.
2. Corrected the input format of the pathway table to pan-Draft. It should not be zipped.
3. Improved the labels in the output table: "pan-reactome_stat.tsv".
4. Added the possibility to save the panGEMs in SBML format by default.
5. Included 'tidyr' as a dependency for installation.

Please let me know if there is anything that needs adjustment.

Ciao,
Nicola

Waschina commented:
Hi Nicola,

ideally, we don't need "tidyr", since we are mainly relying on "data.table". I usually try to avoid using both packages at the same time to avoid potential conflicts and for more code style consistency. Can you point me to the respective code lines, where you use tidyr functions? I can try to replace them with data.table functions.

nicola-debernardini commented:
Hi Silvio,

I reviewed the script and found that "tidyr" was a leftover from previous graphical developments, which has now been removed. I should have checked this earlier. I've removed "tidyr" from pan-draft.R and updated the installation documentation accordingly.

It should be ok now.

Waschina reviewed (approved):
Looks all good. Thank you for the Update & PR!
